### ISSUE TITLE
keep getting emergency sells

### ISSUE THREAD

**Issue by aristosv** [freqtrade/freqtrade#5770]
## Describe your environment

  * Operating system: Debian 10
  * Python Version: Python 3.7.3
  * CCXT version: ccxt==1.58.47
  * Freqtrade Version: freqtrade develop-545446022
  
## Your question

I am using a simple bollinger bands strategy. I just want to buy when BTC drops and sell when it goes back up. But for some reason I only get emergency sells with small losses. These are a few examples.

<img width="279" alt="3" src="https://user-images.githubusercontent.com/528302/138414742-425dc691-d225-4f4a-a183-e8c91ad96066.png">

<img width="285" alt="1" src="https://user-images.githubusercontent.com/528302/138414749-e0470930-a330-46f2-974f-30018e8b138d.png">

<img width="278" alt="2" src="https://user-images.githubusercontent.com/528302/138414762-a2ff6b9c-5315-4b79-8809-ddd3889b1beb.png">

```
from freqtrade.strategy.interface import IStrategy
from pandas import DataFrame
import talib.abstract as ta
import freqtrade.vendor.qtpylib.indicators as qtpylib
class freqtrade_aristos(IStrategy):

    stoploss = -0.99
    timeframe = '15m'
    trailing_stop = True
    trailing_stop_positive = 0.01
    trailing_stop_positive_offset = 0.02
    trailing_only_offset_is_reached = True

    def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
        bollinger = qtpylib.bollinger_bands(qtpylib.typical_price(dataframe), window=20, stds=2)
        dataframe['bb_lowerband'] = bollinger['lower']
        dataframe['bb_middleband'] = bollinger['mid']
        dataframe['bb_upperband'] = bollinger['upper']
        return dataframe

    def populate_buy_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
        dataframe.loc[
            (
                    (dataframe['close'] < dataframe['bb_lowerband'])
            ),
            'buy'] = 1
        return dataframe

    def populate_sell_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
        dataframe.loc[
            (
                    (dataframe['close'] > dataframe['bb_upperband'])
            ),
            'sell'] = 1
        return dataframe
```

**Comment by xmatthias** [comment]
emergency-sells happen only in one instance - if setting the stoploss on exchange fails.

As you're using a stoploss of -0.99, freqtrade tries to place a stoploss-order 99% below your entry. Unless you're using rather high sums to trade (screenshots suggest you're not), this will end you below the minimum tradable of 10$.

With a stoploss of -0.99, you cannot use stoploss on exchange (There's no point, there is no stoploss).
best increase the stoploss, acceptiong a 99% loss will ruin you should it come to that (markets WILL turn at some point).
I'd not feel comfortable with any stoploss bigger than 10% - gaining one bigger stoploss back will take you tons of trades (often 5-10). While a large stoploss will not realize the losses, it'll still lock up the capital, and potentially that coin never raises agian, resulting in a 100% loss.

**Comment by aristosv** [comment]
thanks for the info
